src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:24079/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:24079/project/issues?id=sample&branch=feature3&issues=AWGyw4g_hHBWNRKxQDP3&open=AWGyw4g_hHBWNRKxQDP3)

[ISSUE_KEY]: AWGyw4g_hHBWNRKxQDP3